tools/federation/build/README.txt

```diff
@@ -0,0 +1,15 @@
+Readme
+
+First steps and important information:
+
+ * Documentation can be found at https://github.com/VeritasOS/fedx/wiki
+ * Required libraries are in the lib folder
+ * Github Page: https://github.com/VeritasOS/fedx
+ * Contact: Andreas.Schwarte (at) veritas.com / aschwarte10 (at) gmail.com
+ * Project Website: https://github.com/VeritasOS/fedx
+ * FedX is licensed under Apache License 2.0 (see attached license). 
```

Comment:
I'm guessing this file should be changed or removed?

Reply:
Let's remove it, and add a section on the history of FedX in the documentation later.